README.md

```diff
@@ -20,21 +20,11 @@ v1.23.0 which appends random strings to container names each time they run.
 1. Install Docker and add your user to the Docker group 
    (e.g. https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04.) 
    You may need to restart your machine for group changes to take effect.
-1. Configure your Docker client to use our registry by following instructions 
-   here: https://github.com/vimc/montagu-registry/tree/master#login 
-   
-   This is where you will need to have Vault 
-   installed, and the VAULT_ADDR variable specified, e.g. by adding 
-   `export VAULT_ADDR='https://support.montagu.dide.ic.ac.uk:8200'` to your profile using ` sudo nano ~/.profile` 
 1. Install node dependencies by running `npm install --prefix src/app/static`. Javascript and CSS will be compiled automatically as part of the gradle build.
+1. For local development and testing run all dependencies (Orderly Server etc.) with `./dev/run-dependencies.sh`.
```

Comment:
While we're housekeeping, it's probably worth adding here that you'll also want to run the add-test-users script in order to have access to the demo reports (or maybe just roll running that script into dev/run-dependencies?) and mention the login details.

Reply:
Well spotted, thanks